### DISCUSSION TITLE
Nuxt3 SSG : Can generated NuxtLink include the full path to the html file ?

### DISCUSSION THREAD

**Discussion by Pic212** [nuxt/nuxt#22204] [Questions] [answered]
Hi, 
Newbie question .. :)

Is it possible when generating a static website to have the full path to the html file in the generated link ?

I have this link  `<NuxtLink to="/about/organization">About page</NuxtLink>` (in the Folder pages, a subfolder about, and inside this about folder a file organization.vue
The generated link ( npm run generate ) is : `<a href="/about/organization" class=""> About page </a>`

I would like to have `<a href="/about/organization/index.html" class=""> About page </a>`

Thanks for your help ! Serge

**Comment by xjccc** [comment, marked as answer]
you can rename your `.vue` file to `.html.vue`
then you can use de correct link

**Reply by Pic212** [reply]
Thanks !!!

**Reply by QimiaoCloud** [reply]
I don't think that's the right answer. 

If you look at "dist" carefully after SSG, the directory structure usually looks like this: somepage/index.html,
Prerender generates a payload for each route (Default), and the payload prefetch in the source of the page , something like "/somepage.html/_payload.js."

I guess it use for hydrant.But the somepage.html directory doesn't really exist, so instead of changing the file name to "somepage.html.vue", you should add an alias to the meta of the page file "somepage.vue" 

definePageMeta({
    alias:['/somepage/index.html','/somepage.html'],
  })

essentially an alias for a VueRouter route. In this way, nuxtlink to "/somepage/index.html" or "/somepage.html" can be matched.
